util/wrap.pl.in

```diff
@@ -47,29 +47,13 @@ local $ENV{OPENSSL_MODULES} = $std_providers
 local $ENV{OPENSSL_CONF} = $std_openssl_conf
     if ($ENV{OPENSSL_CONF} // '') eq '' && -f $std_openssl_conf;
 {-
-     # For VMS, we define logical names to get the libraries properly
-     # defined.
      use File::Spec::Functions qw(rel2abs);
```

Comment:
Is this still needed or can the entire thing enclosed in `{-` and `-}` go away?

Reply:
the entire thing can go away.